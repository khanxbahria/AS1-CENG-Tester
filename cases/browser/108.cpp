#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("ee", "lll","u");
Tab B = Tab("j", "www","sss");
Tab C = Tab("f", "ll","vvv");
Tab D = Tab("aa", "eee","ll");
Tab E = Tab("oo", "dd","rrr");
Tab F = Tab("yy", "rrr","lll");
Tab G = Tab("k", "n","jjj");
Tab H = Tab("ppp", "kk","ccc");
Tab I = Tab("uu", "f","hh");
Tab J = Tab("e", "bbb","cc");
Tab K = Tab("ee", "fff","jjj");
Tab L = Tab("hhh", "ttt","eee");
Tab M = Tab("tt", "ee","c");
Tab N = Tab("ss", "yy","rr");
Tab O = Tab("s", "zzz","iii");
Tab P = Tab("sss", "e","qqq");
Tab Q = Tab("mm", "sss","u");
Tab R = Tab("x", "l","qq");
Tab S = Tab("k", "ss","nnn");
Tab T = Tab("yyy", "a","lll");
Tab U = Tab("d", "jj","vvv");
Tab V = Tab("ll", "uuu","dd");
Tab W = Tab("ff", "j","s");
Tab X = Tab("qqq", "q","gg");
Tab Y = Tab("zzz", "b","pp");
Tab Z = Tab("sss", "p","xx");
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(0);
b.newTab(A);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(0);
b.newTab(B);
c.newTab(C);
d.newTab(D);
a.newTab(E);
b.newTab(F);
c.changeActiveTabTo(3);
d.newTab(G);
a.moveActiveTabTo(0);
b.newTab(H);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.newTab(I);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(J);
d.moveActiveTabTo(2);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.newTab(K);
a.newTab(L);
b.changeActiveTabTo(3);
c.newTab(M);
d.newTab(N);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.newTab(O);
d.newTab(P);
a.changeActiveTabTo(5);
b.newTab(Q);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(0);
b.newTab(R);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(S);
d.newTab(T);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(5);
b.newTab(U);
c.newTab(V);
d.changeActiveTabTo(5);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(3);
b.newTab(W);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.newTab(X);
b.newTab(Y);
c.changeActiveTabTo(4);
d.newTab(Z);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
c.moveActiveTabTo(2);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
c.moveActiveTabTo(0);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.moveActiveTabTo(4);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.moveActiveTabTo(3);
d.changeActiveTabTo(3);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->switchToWindow(6);
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->closeEmptyWindows();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->closeWindow();
brows->print();
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(1);
brows->newWindow();
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->switchToWindow(6);
brows->closeWindow();
brows->getWindow(2);
brows->switchToWindow(6);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->newWindow();
brows->getWindow(4);
brows->newWindow();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(3);
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(4);
brows->mergeAllWindows();
brows->switchToWindow(0);
brows->switchToWindow(7);
brows->switchToWindow(9);
brows->closeWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->switchToWindow(7);
brows->switchToWindow(8);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(3);
brows->print();
brows->switchToWindow(0);
brows->getWindow(6);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->closeEmptyWindows();
brows->print();
brows->switchToWindow(2);
brows->print();
brows->newWindow();
brows->switchToWindow(1);
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(8);
brows->getWindow(7);
brows->closeEmptyWindows();
brows->switchToWindow(7);
brows->switchToWindow(4);
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->switchToWindow(5);
brows->closeEmptyWindows();
brows->closeWindow();
brows->switchToWindow(9);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(5);
brows->mergeAllWindows();
brows->newWindow();
brows->getWindow(3);
brows->switchToWindow(9);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->switchToWindow(6);
brows->getWindow(0);
brows->switchToWindow(2);
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->closeEmptyWindows();
brows->switchToWindow(3);
brows->closeEmptyWindows();
delete brows;
}
	return 0;
}