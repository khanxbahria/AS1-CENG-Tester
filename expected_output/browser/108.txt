The active tab is tab 0




The active tab is tab 3
ttt
hhh
eee

dd
oo
rrr

q
qqq
gg





The active tab is tab 3
www
j
sss

lll
ee
u

b
zzz
pp

l
x
qq

j
ff
s





jj
d
vvv

sss
mm
u

rrr
yy
lll

kk
ppp
ccc

The active tab is tab 5
zzz
s
iii

ll
f
vvv

ss
k
nnn

uuu
ll
dd

bbb
e
cc





ee
tt
c

The active tab is tab 5
a
yyy
lll

p
sss
xx

yy
ss
rr

f
uu
hh





eee
aa
ll

fff
ee
jjj

e
sss
qqq

n
k
jjj

The active tab is tab 0




The active tab is tab 3
ttt
hhh
eee

dd
oo
rrr

q
qqq
gg





The active tab is tab 3
www
j
sss

lll
ee
u

b
zzz
pp

l
x
qq

j
ff
s





jj
d
vvv

sss
mm
u

rrr
yy
lll

kk
ppp
ccc

The active tab is tab 5
zzz
s
iii

ll
f
vvv

ss
k
nnn

uuu
ll
dd

bbb
e
cc





ee
tt
c

The active tab is tab 5
a
yyy
lll

p
sss
xx

yy
ss
rr

f
uu
hh





eee
aa
ll

fff
ee
jjj

e
sss
qqq

n
k
jjj

The active tab is tab 3
ttt
hhh
eee

dd
oo
rrr

q
qqq
gg





The active tab is tab 3
www
j
sss

lll
ee
u

b
zzz
pp

l
x
qq

j
ff
s





jj
d
vvv

sss
mm
u

rrr
yy
lll

kk
ppp
ccc

The active tab is tab 5
zzz
s
iii

ll
f
vvv

ss
k
nnn

uuu
ll
dd

bbb
e
cc





ee
tt
c

The active tab is tab 5
a
yyy
lll

p
sss
xx

yy
ss
rr

f
uu
hh





eee
aa
ll

fff
ee
jjj

e
sss
qqq

n
k
jjj

The active tab is tab 26




www
j
sss

lll
ee
u

b
zzz
pp

l
x
qq

j
ff
s





jj
d
vvv

sss
mm
u

rrr
yy
lll

kk
ppp
ccc

zzz
s
iii

ll
f
vvv

ss
k
nnn

uuu
ll
dd

bbb
e
cc





ee
tt
c

a
yyy
lll

p
sss
xx

yy
ss
rr

f
uu
hh





eee
aa
ll

fff
ee
jjj

e
sss
qqq

n
k
jjj

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 3
















Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 1








The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The browser is empty
